src/components/AgreementLineSections/Coverage/Coverage.test.js

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import '@folio/stripes-erm-components/test/jest/__mock__';
+import { renderWithIntl } from '@folio/stripes-erm-components/test/jest/helpers';
+import { Accordion, MultiColumnList, MultiColumnListCell, KeyValue } from '@folio/stripes-testing';
+import { line, resource, defaultLine, defaultResource } from './testResources';
+import translationsProperties from '../../../../test/helpers';
+import Coverage from './Coverage';
+
+describe('Coverage', () => {
+    describe('renders with coverage type custom ', () => {
+      beforeEach(() => {
+        renderWithIntl(
+          <Coverage line={line} resource={resource} />,
+          translationsProperties
+        );
+      });
+
+      test('renders Coverage Accordion', async () => {
+        await Accordion('Coverage').exists();
+      });
+
+      test('renders Embargo Accordion', async () => {
```

Comment:
Description says Accordion but the test is checking for a KeyValue?

Reply:
I fixed that